flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/PredefinedOptions.java

```diff
@@ -61,7 +61,9 @@
         public DBOptions createDBOptions(Collection<AutoCloseable> handlesToClose) {
             return new DBOptions()
                     .setUseFsync(false)
-                    .setInfoLogLevel(InfoLogLevel.HEADER_LEVEL)
+                    .setInfoLogLevel(InfoLogLevel.INFO_LEVEL)
+                    .setMaxLogFileSize(DEFAULT_LOG_FILE_SIZE)
+                    .setKeepLogFileNum(DEFAULT_LOG_FILE_NUM)
```

Comment:
Mind explain why we don't set these default values in `RocksDBConfigurableOptions` directly? Thanks

Reply:
@NicoK Thanks for your feedback, after discussion with @carp84 offline, I decide to hold this PR currently and create another ticket https://issues.apache.org/jira/browse/FLINK-24046 to track the refactor.